prototyping/mocking/mockSearch.cpp

```diff
@@ -0,0 +1,48 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+void search(unordered_set<string> words, vector<string> articles) {
+	unordered_map<string, int> incidences;
+
+	for (const auto & article : articles) {
+		string curr_word = "";
+		
+		for (const auto & letter : article) {
```

Comment:
Have you and @jaimehisao had a chance to sync on the inputs/outputs of the parser?

It might be best to have the parser component to handle indexing the articles (creating a map from word to number of occurrences of a word in the article) during the initial parse of the article, since that never changes and can be done during the first pass.

Reply:
We haven't discussed that yet but it is definitely something that we need to do and define soon. If @Domene99 is okay with the parser doing the indexing I can make that happen.

Reply:
Yeah, I simply assumed that I'd receive an array of articles, which now is an array of jsons with the article full text as a value. Correct me if I'm wrong @jaimehisao

Reply:
And in the mvp we can discuss creating a frequency map for every article

Reply:
Yes, the array of articles was the original plan. Sure, we can discuss that @Domene99